data/json/mapgen/outpost.json

```diff
@@ -0,0 +1,244 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "outpost" ],
+    "object": {
+      "weight": 100,
+      "rows": [
+        "|----------++----------|",
+        "|x         xx         x|",
+        "|   //////    /////    |",
+        "|   /b.lb//=///&/&/    |",
```

Comment:
It seems to me the exterior doors of a base like this would probably be metal?

Reply:
Well, it could be, of course. I just looked at the original, where exterior doors are wooden.
I'll see on this more carefully tomorrow.